.eslintrc

```diff
@@ -25,7 +25,6 @@
       "version": "detect"
     }
   },
-  "ignorePatterns": ["mocks/**", "coverage/**", "next.config.js"],
```

Comment:
Yeah good point - I actually removed this because we have a .eslintignore and these are not really patterns. So I've made sure coverage and mocks is in that file (now) just to avoid confusion.